### ISSUE TITLE
Revoking a key

### ISSUE THREAD

**Issue by gitluv** [keybase/keybase-issues#2214]
Is it possible to revoke a... revoked key on Keybase? I thought starting fresh and adding my new key would delete/hide/strikethrough everything pertaining to and including to my revoked key on the sigchain, but it all still there with nothing to suggest a key revocation. Did I do something wrong? I used the web interface.

I found this https://keybase.io/docs/api/1.0/call/key/revoke but I thought I'd ask anyway.

**Comment by maxtaco** [comment]
A sigchain reset is different from a key revocation.  A sigchain reset is akin to starting all over with a new username, except we let you reuse the username.  Everything else is considered out of use.

What's your username?

**Comment by gitluv** [comment]
Right, but should the sigchain reset when I delete my old key and start over? Id: keithr

**Comment by maxtaco** [comment]
Looks like you've reset your account twice now.  Currently, your sigchain is reset, and you don't have a public key published in the [sitewide merkle tree](https://keybase.io/_/api/1.0/merkle/path.json?uid=f3abb0405c131d157dd12a6b34d29219).  You can now start your sigchain again with a new key or an existing key.

BTW, there could be bugs in the [sigchain](https://keybase.io/keithr/sigchain) view of your account. The [raw version](https://keybase.io/keithr/chain) is always authoritative.

**Comment by maxtaco** [comment]
cc: @malgorithms, there might be further bugs with the sigchain view on account reset...
